electronicparsers/openmx/parser.py

```diff
@@ -732,17 +731,16 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
             name='OpenMX', version=str(mainfile_parser.get('program_version'))
         )
         date_start = mainfile_parser.get('date_start')
-
-        elapsed_time = mainfile_parser.get('elapsed_time')
-
         date_start_datetime = datetime.strptime(
                     date_start, '%a %b %d %H:%M:%S %Y')
         date_start_timestamp = date_start_datetime.timestamp()
-        date_end = date_start_timestamp + elapsed_time
 
-        sec_run.time_run = TimeRun()
-        sec_run.time_run.date_end = date_end
-        sec_run.time_run.date_start = date_start_timestamp
+        elapsed_time = mainfile_parser.get('elapsed_time')
+        if elapsed_time is not None:
+            date_end = date_start_timestamp + elapsed_time
+            sec_run.time_run = TimeRun()
+            sec_run.time_run.date_end = date_end
+            sec_run.time_run.date_start = date_start_timestamp
```

Comment:
Actually, `sec_run.time_run = TimeRun()` and `sec_run.time_run.date_start = date_start_timestamp` should not be in the if block, they make sense even if we fail to parse the elapsed time.